pros/src/error.rs

```diff
@@ -98,9 +122,14 @@ pub enum PortError {
         "The port you specified couldn't be configured as what you specified."
     ))]
     PortCannotBeConfigured,
+    #[snafu(display("Unknown error code {num}!"))]
+    Unknown { num: i32 },
```

Comment:
Any reason why this shouldn't be a tuple variant?

Reply:
Yes, it's not supported by the Snafu derive macro

Reply:
Ah.